assets/templates/kubernetes.container/gotext_multiline.tpl

```diff
@@ -0,0 +1,554 @@
+{{- $period := generate "metricset.period" }}
+{{- $timestamp := generate "timestamp" }}
+{{- $agentId := generate "agent.id" }}
+{{- $agentVersion := generate "agent.version" }}
+{{- $agentName := generate "agent.name" }}
+{{- $agentEphemeralid := generate "agent.ephemeral_id" }}
+{{- $rxbytes := generate "container.network.ingress.bytes" | int }}
+{{- $txbytes := generate "container.network.egress.bytes" | int }}
+{{- $uId := uuidv4 }}
+{{- $pod_uId := uuidv4 }}
+{{- $container_uId := uuidv4 }}
+{{- $suffix := split "-" $uId }}
+{{- $offset := generate "Offset" | int }}
+{{- $faults := generate "faults" | int }}
+{{- $pct := generate "Percentage" | float64 }}
+{  "@timestamp": "{{$timestamp.Format "2006-01-02T15:04:05.999999Z07:00"}}",
+   "container":{
+      "memory":{
+         "usage": {{divf $pct 1000000}}
+      },
+      "name":"{{ generate "container.name"}}",
+      "runtime":"containerd",
+      "cpu":{
+         "usage": {{divf $pct 1000000}}
+      },
+      "id":"{{ $container_uId }}"
+   },
+   "kubernetes": {
+      "container":{
+         "start_time":"{{$timestamp.Format "2006-01-02T15:04:05.999999Z07:00"}}",
+         "memory":{
+            "rss":{
+               "bytes": {{generate "Bytes"}}
+            },
+            "majorpagefaults": {{ $faults }},
+            "usage":{
+               "node":{
+                  "pct": {{divf $pct 1000000}}
+               },
+               "bytes": {{generate "Bytes"}},
+               "limit":{
+                  "pct": {{divf $pct 1000000}}
+               }
+            },
+            "available":{
+               "bytes": {{generate "Bytes"}}
+            },
+            "workingset":{
+               "bytes": {{generate "Bytes"}},
+               "limit":{
+                  "pct": {{divf $pct 1000000}}
+               }
+            },
+            "pagefaults": "{{ $faults }}"
+         },
+         "rootfs":{
+            "inodes":{
+               "used": {{ generate "kubernetes.container.rootfs.inodes.used" }}
+            },
+            "available":{
```

Comment:
`{{generate "Bytes"}}`

not sure if it's a concern for you, but I'd guess that the sum of the various `rootfs` stats will not be "correct" (meaning used+available not matching capacity or similar)

Reply:
Yes no concern at the moment